tests/test/check/data/main.fmf

```diff
@@ -19,6 +19,7 @@
     test: |
       set -x
       systemctl status auditd || /bin/true
+      systemctl start auditd
```

Comment:
Shouldn't we run this during the before-test `avc` check instead?

Reply:
I'm not sure. We definitely want to check whether `auditd` is running, I'm less sure whether the check should start it. It might have been stopped by the test, for a reason. As of now, I think the test should detect its state, and report either `skip` or `error` if it's not running, plus it should be mentioned in docs that `auditd` needs to run.

In any case, I plan to do these changes to AVC check in a separate PR, this one I wanted to focus on making the test work. It may turn out `start auditd` would remain here, as "users' responsibility", or it would move into the check as you suggested, and I'd remove it from this file, but it would be under the "making AVC check deal with auditd not running" label rather than "making the AVC check test work on rawhide again" :)

Reply:
FYI, if auditd is not running then AVCs are logged into the kernel ring buffer or system log. If AVC detection would be based only on `/var/log/audit/audit.log` then I would propose so have this limitation documented (maybe it already is).

Reply:
Indeed, it shall be documented. The detection is based on running `ausearch`, which depends on `auditd` as we learned recently. I will add a note to the documentation in the PR improving the test.

Reply:
Yeah, makes sense. Thanks for fixing this!